sklearn/tests/test_random_projection.py

```diff
@@ -367,6 +380,7 @@ def test_johnson_lindenstrauss_min_dim():
 
 @pytest.mark.parametrize("random_projection_cls", all_RandomProjection)
 def test_random_projection_feature_names_out(random_projection_cls):
+    data, _ = make_sparse_random_data(sp.coo_array, n_samples, n_features, n_nonzeros)
```

Comment:
In particular this call needs to be updated.

Reply:
I did parametrize this test now . In this test, the data made with both datatypes from the COO container is created, fitted and then `_feature_names_out` are compared to the expected feature names.

Though, @glemaitre was suggesting maybe not to parametrize here, did I get this right? If so, should I put `make_sparse_random_data(sp.coo_matrix, n_samples, n_features, n_nonzeros)` instead?